src/main/java/frc/robot/commands/ControlPanelCommand.java

```diff
@@ -0,0 +1,35 @@
+package frc.robot.commands;
+
+import com.revrobotics.ColorMatchResult;
+import com.ctre.phoenix.motorcontrol.VictorSPXControlMode;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj.util.Color;
+
+import frc.robot.subsystems.ControlPanelSubsystem;
+
+public class ControlPanelCommand extends CommandBase {
+
+    ControlPanelSubsystem controlPanel;
+
+    public ControlPanelCommand() {
+        controlPanel = new ControlPanelSubsystem();
```

Comment:
You might also need a call to `addRequirements()`, though I'm not certain if it's required